scripts/install.sh

```diff
@@ -295,14 +324,37 @@ if ! grep -Fq "devAwsUserAccountArn" serverless_config.json; then
     echo -e "{\n  \"devAwsUserAccountArn\": \"$IAMUserARN\"\n}" >> serverless_config.json
 fi
 
+if [[ $multiTenancyEnabled == true ]]; then
+    mtEnabledArgs=(--useMultiTenancy "true")
+else
+    mtEnabledArgs=()
+fi
+
+if [[ $multiTenancyEnabled == true && "$multiTenancyTokenClaim" != "" ]]; then
```

Comment:
* This if should be moved into previous `if [[ $multiTenancyEnabled == true ]]` so the `$multiTenancyEnabled == true` check will not be needed
* `"$multiTenancyTokenClaim" != ""` should be `-n "$multiTenancyTokenClaim"` (idiomatic)
* Can we maybe use `mtArgs` (new) instead of all of these: `mtEnabledArgs`, `mtTokenArgs`, `mtTenantTypeSubUrlArgs`? I think it should simplify.